Javadocs fail to build on JDK 17

## Expected Behavior

```gradle build```
should run without errors.


## Actual Behavior

The following errors are reported from the javadoc task:

```
> Task :richtextfx:javadoc
warning: URL http://docs.oracle.com/javase/8/docs/api/element-list was redirected to https://docs.oracle.com/en/java/javase/19/ -- Update the command-line options to suppress this warning.
warning: URL http://docs.oracle.com/javase/8/javafx/api/element-list was redirected to https://docs.oracle.com/en/java/javase/19/ -- Update the command-line options to suppress this warning.
/Users/scott/dev/opensource/RichTextFX/richtextfx/src/main/java/org/fxmisc/richtext/GenericStyledArea.java:104: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Adding Scrollbars to the Area</h3>
   ^
/Users/scott/dev/opensource/RichTextFX/richtextfx/src/main/java/org/fxmisc/richtext/MultiChangeBuilder.java:17: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Relative vs Absolute Changes</h3>
   ^
/Users/scott/dev/opensource/RichTextFX/richtextfx/src/main/java/org/fxmisc/richtext/TextEditingArea.java:440: error: unexpected heading used: <H3>, compared to implicit preceding heading: <H3>
     * <h3>Warning: Off-By-One errors can easily occur</h3>
       ^
/Users/scott/dev/opensource/RichTextFX/richtextfx/src/main/java/org/fxmisc/richtext/model/StyledDocument.java:172: error: unexpected heading used: <H3>, compared to implicit preceding heading: <H3>
     * <h3>Warning: Off-By-One errors can easily occur</h3>
       ^
/Users/scott/dev/opensource/RichTextFX/richtextfx/src/main/java/org/fxmisc/richtext/StyledTextField.java:218: error: tag not supported in HTML5: tt
     * The prompt text to display or <tt>null</tt> if no prompt text is to be displayed.
                                     ^
5 errors
```

## Environment info:

- RichTextFX Version: commit 8982fd0dc7b7a13f076ca5696f3337ee9e6d5e15
- Operating System: macOS 13.1
- Java version: 17.0.5

--- a/build.gradle
+++ b/build.gradle
@@ -20,3 +20,9 @@ subprojects {
     }
 
 }
+
+allprojects {
+   tasks.withType(Javadoc) {
+      options.addStringOption("Xmaxwarns", "1")
+   }
+}

--- a/richtextfx/build.gradle
+++ b/richtextfx/build.gradle
@@ -71,8 +71,8 @@ javadoc {
 
     options.links = [
         // resolve links to Java and JavaFX Javadocs
-        'http://docs.oracle.com/javase/8/docs/api/',
-        'http://docs.oracle.com/javase/8/javafx/api/',
+        'https://docs.oracle.com/en/java/javase/11/docs/api/',
+        'https://openjfx.io/javadoc/11/',
 
         // resolve links to ReactFX
         'https://www.javadoc.io/doc/org.reactfx/reactfx/2.0-M5/',

--- a/richtextfx/src/main/java/org/fxmisc/richtext/GenericStyledArea.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/GenericStyledArea.java
@@ -101,7 +101,7 @@ import org.reactfx.value.Var;
  * Accepts user input (keyboard, mouse) and provides API to assign style to text ranges. It is suitable for
  * syntax highlighting and rich-text editors.
  *
- * <h3>Adding Scrollbars to the Area</h3>
+ * <H3>Adding Scrollbars to the Area</H3>
  *
  * <p>By default, scroll bars do not appear when the content spans outside of the viewport.
  * To add scroll bars, the area needs to be wrapped in a {@link VirtualizedScrollPane}. For example, </p>
@@ -116,7 +116,7 @@ import org.reactfx.value.Var;
  * parent.getChildren().add(vsPane)
  * </code></pre>
  *
- * <h3>Auto-Scrolling to the Caret</h3>
+ * <H3>Auto-Scrolling to the Caret</H3>
  *
  * <p>Every time the underlying {@link EditableStyledDocument} changes via user interaction (e.g. typing) through
  * the {@code GenericStyledArea}, the area will scroll to insure the caret is kept in view. However, this does not
@@ -147,7 +147,7 @@ import org.reactfx.value.Var;
  * <p>Additionally, when overriding the default user-interaction behavior, remember to include a call
  * to {@link #requestFollowCaret()}.</p>
  *
- * <h3>Setting the area's {@link UndoManager}</h3>
+ * <H3>Setting the area's {@link UndoManager}</H3>
  *
  * <p>
  *     The default UndoManager can undo/redo either {@link PlainTextChange}s or {@link RichTextChange}s. To create
@@ -155,7 +155,7 @@ import org.reactfx.value.Var;
  *     with others), create them using the convenient factory methods in {@link UndoUtils}.
  * </p>
  *
- * <h3>Overriding default keyboard behavior</h3>
+ * <H3>Overriding default keyboard behavior</H3>
  *
  * {@code GenericStyledArea} uses {@link javafx.scene.input.KeyEvent#KEY_TYPED KEY_TYPED} to handle ordinary
  * character input and {@link javafx.scene.input.KeyEvent#KEY_PRESSED KEY_PRESSED} to handle control key
@@ -179,7 +179,7 @@ import org.reactfx.value.Var;
  * Nodes.addInputMap(area, consume(keyPressed(S, CONTROL_DOWN), event -&gt; save()));
  * </code></pre>
  *
- * <h3>Overriding default mouse behavior</h3>
+ * <H3>Overriding default mouse behavior</H3>
  *
  * The area's default mouse behavior properly handles auto-scrolling and dragging the selected text to a new location.
  * As such, some parts cannot be partially overridden without it affecting other behavior.
@@ -250,14 +250,14 @@ import org.reactfx.value.Var;
  *     </li>
  * </ul>
  *
- * <h3>CSS, Style Classes, and Pseudo Classes</h3>
+ * <H3>CSS, Style Classes, and Pseudo Classes</H3>
  * <p>
  *     Refer to the <a href="https://github.com/FXMisc/RichTextFX/wiki/RichTextFX-CSS-Reference-Guide">
  *         RichTextFX CSS Reference Guide
  *     </a>.
  * </p>
  *
- * <h3>Area Actions and Other Operations</h3>
+ * <H3>Area Actions and Other Operations</H3>
  * <p>
  *     To distinguish the actual operations one can do on this area from the boilerplate methods
  *     within this area (e.g. properties and their getters/setters, etc.), look at the interfaces
@@ -267,7 +267,7 @@ import org.reactfx.value.Var;
  *     To update multiple portions of the area's underlying document in one call, see {@link #createMultiChange()}.
  * </p>
  *
- * <h3>Calculating a Position Within the Area</h3>
+ * <H3>Calculating a Position Within the Area</H3>
  * <p>
  *     To calculate a position or index within the area, read through the javadoc of
  *     {@link org.fxmisc.richtext.model.TwoDimensional} and {@link org.fxmisc.richtext.model.TwoDimensional.Bias}.

--- a/richtextfx/src/main/java/org/fxmisc/richtext/MultiChangeBuilder.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/MultiChangeBuilder.java
@@ -14,7 +14,7 @@ import java.util.List;
  * {@link org.fxmisc.richtext.model.EditableStyledDocument} in one call via {@link #commit()}. <strong>Note: this builder
  * cannot be reused and a new one must be created for each multi-change.</strong>
  *
- * <h3>Relative vs Absolute Changes</h3>
+ * <H3>Relative vs Absolute Changes</H3>
  * <p>
  *     Let's say that a document has the text {@code |(|t|e|x|t||)|} where each {@code |} represents a position
  *     in-between the characters of the text . If one wants to remove the opening
@@ -79,7 +79,7 @@ import java.util.List;
  *     do not. To make things easier for the developer, the methods declared here are similar to those declared
  *     in {@link EditActions}, minus a few (e.g. {@link EditActions#append(StyledDocument)}).
  * </p>
- * <h3>Other Considerations</h3>
+ * <H3>Other Considerations</H3>
  * <ul>
  *     <li>
  *         <strong>Warning: </strong> relative changes will not be updated if one starts a multi-change and fails

--- a/richtextfx/src/main/java/org/fxmisc/richtext/StyledTextField.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/StyledTextField.java
@@ -215,7 +215,7 @@ public abstract class StyledTextField<PS, S> extends StyledTextArea<PS, S>
 
 
     /**
-     * The prompt text to display or <tt>null</tt> if no prompt text is to be displayed.
+     * The prompt text to display or {@code null} if no prompt text is to be displayed.
      * <p>The Text will be aligned according to the text fields alignment setting and have a default
      * text fill of GRAY unless you have changed it by any means, e.g. with CSS "-fx-prompt-text-fill" 
      */

--- a/richtextfx/src/main/java/org/fxmisc/richtext/TextEditingArea.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/TextEditingArea.java
@@ -437,7 +437,7 @@ public interface TextEditingArea<PS, SEG, S> {
      * └ position 0
      * </pre>
      *
-     * <h3>Warning: Off-By-One errors can easily occur</h3>
+     * <H3>Warning: Off-By-One errors can easily occur</H3>
      * <p>If the column index spans outside of the given paragraph's length, the returned value will
      * pass on to the previous/next paragraph. In other words, given a document with two paragraphs
      * (where the first paragraph's text is "some" and the second "thing"), then the following statements are true:</p>

--- a/richtextfx/src/main/java/org/fxmisc/richtext/model/StyledDocument.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/model/StyledDocument.java
@@ -169,7 +169,7 @@ public interface StyledDocument<PS, SEG, S> extends TwoDimensional {
      * └ position 0
      * </pre>
      *
-     * <h3>Warning: Off-By-One errors can easily occur</h3>
+     * <H3>Warning: Off-By-One errors can easily occur</H3>
      * <p>If the column index spans outside of the given paragraph's length, the returned value will
      * pass on to the previous/next paragraph. In other words, given a document with two paragraphs
      * (where the first paragraph's text is "some" and the second "thing"), then the following statements are true:</p>

